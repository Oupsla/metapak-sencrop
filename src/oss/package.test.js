"use strict";

const packageTransformer = require("./package");

describe("Main", () => {
  describe("Package transformer", () => {
    test("should work with an empty package.json", () => {
      expect(packageTransformer({})).toMatchSnapshot();
    });
  });
});
