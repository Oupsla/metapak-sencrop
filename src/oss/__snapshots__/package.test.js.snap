// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Package transformer should work with an empty package.json 1`] = `
Object {
  "license": "MIT",
}
`;
